src/Encoder.ts

```diff
@@ -300,14 +301,11 @@ export class Encoder {
     this.pos++;
   }
 
-  writeU8v(...values: Array<number>) {
+  writeU8a(values: ArrayLike<number>) {
```

Comment:
YES, I was not spend too much time in Encoder, but i was going to mention to you ...reset params usage. This is not good for performance. TS maps it into loops and additional array allocation from passed arguments.

Reply:
Yeah, it is not just slow but causes RangeError for a large array. Too bad.